src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,39 +1,54 @@
 import { Breadcrumbs as MuiBreadcrumbs } from '@mui/material';
 
 import { Svg } from 'components/Svg';
-import { variables } from 'styles/variables';
-import { StyledTitleSmall } from 'styles/styledComponents/Typography';
 import { breadcrumbs } from 'redux/modules';
+import { StyledLabelSmall, StyledTitleSmall } from 'styles/styledComponents/Typography';
+import { variables } from 'styles/variables';
 
-import { StyledAvatarWrapper, StyledAvatar, StyledLink, StyledBox } from './Breadcrumbs.styles';
+import {
+  StyledLink,
+  StyledBox,
+  StyledIconWrapper,
+  StyledIconImg,
+  StyledPlaceholder,
+} from './Breadcrumbs.styles';
 
 export const Breadcrumbs = (): JSX.Element => {
   const breadcrumbsData = breadcrumbs.useData();
-  const getBreadcrumbAvatar = (index: number) => {
-    if (index === 0) {
-      return <Svg id="workspace" width="3.2rem" height="3.2rem" />;
+
+  const getBreadcrumbIcon = (icon: string | JSX.Element, label: string) => {
+    if (typeof icon === 'string') {
+      icon
+        ? (icon = <StyledIconImg src={icon} alt="Icon" />)
```

Comment:
it's better to avoid mutations of functions arguments. pls create variable for resulted icon

Reply:
done